MemberExpectations.md

```diff
@@ -0,0 +1,63 @@
+It is important that members of our leadership groups (TSC, CommComm) act
+in a way that not only complies with the
+[Code of Conduct](https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md)
+but that supports the vision, mission and operating principles of the Node.js project.
+
+Members are encouraged to have individual opinions and are encouraged
+to express those opinions during discussions within the organization. While
+discourse is encouraged internally, members also have the obligation outside
+of the organization (for example in social media) to be respectful of the
+decisions made within the projects's documented processes.  A member who does
```

Comment:
I would put this differently: Members have an obligation to be respectful of the documented processes they voluntarily agreed to follow, including the results of those processes even (and especially) if the results run counter to their personal point of view. In other words, if you've agreed in advance to the process, you have an obligation to abide by it.